buildflow/runtime/processor.py

```diff
@@ -1,14 +1,17 @@
-import inspect
 from typing import Optional
 
 from buildflow import utils
-from buildflow.api import ProcessorAPI, AutoscalingOptions
-from buildflow.api.io import SinkType, SourceType
+from buildflow.api import (AutoscalingOptions, ProcessorAPI, SinkType,
+                           SourceType)
 from buildflow.runtime import Runtime
 from buildflow.runtime.ray_io import empty_io
 
 
 class Processor(ProcessorAPI):
+
+    def __init__(self, name: str = "") -> None:
```

Comment:
add args & kwargs

Reply:
just kwargs actually

Reply:
change again, anything outside api should explicitly set options. anything optional must be set with a kwarg